Allow setting amount in cents

avdgaag opened the pull request:
As per https://github.com/wvanbergen/clieop/blob/master/lib/clieop/payment/batch.rb#L62, the gem currently seems to require you to pass in money as a Float. It seems to me that the library should not force this decision on the caller. My application happens to store money in cents, and now I am forced to convert my cents to euros, because the Clieop gem insists to conver the  amount to cents.

Then again, it _is_ a nice convenience function for when you do want to pass in an amount in euros.

Would it possible to allow both? Maybe allow passing in options `:amount_in_cents` or `:amount_in_euros`, so the caller can choose to have its value converted or not?

Maybe you could do something like this to allow the value in cents and defaulting to the old behaviour in euros:

``` ruby
amount_in_cents  = tr.fetch(:amount_in_cents) { (tr[:amount] * 100).round.to_i }
```

wvanbergen commented:
This would work for me; it makes sense to have a cents option that doesn't need to rely on rounding.
However, it should be backwards compatible with the current way of doing things.

Commit:
Allow setting amount_in_cents

When generating a batch allow passing in a `:amount_in_cents` option to
`add_transaction` so the gem will not try to convert euros (or dollars)
to cents by multiplying by 100 and rounding.

When no `:amount_in_cents` is given, default to the old behaviour.

avdgaag commented:
I have implemented this feature based on my earlier example. Please review if this is sufficient.

wvanbergen commented:
I'd say :shipit:
@moiristo Any feedback?

moiristo commented:
nope, looks good.
